core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -584,6 +585,11 @@ protected void onModified() throws IOException {
      */
     /*package*/ int scmCheckoutRetryCount;
 
+    /**
+     * For {@link hudson.slaves.ComputerRetentionWork#getRecurrencePeriod()}
+     */
+    /*package*/ int computerRetentionCheckInterval = 60;
```

Comment:
I _think_ Jenkins has a special deserialization behavior, but it'd be safer if there's a test asserting that an existing Jenkins home will not set this to 0 on startup.

Reply:
Done via c962e36afa6e6af051d8cfde2436424019fcb65f. I've also added a 60s fallback to the getter when the serialized value is not greater than zero.